### ISSUE TITLE
CNTK RC1 - Brainscript/CNTK issues

### ISSUE THREAD

**Issue by avader906** [Microsoft/CNTK#1764]
**1) Display issue - incorrect Epoch completion percentages**

When the training is restarted with makeMode = true (Brainscipt model / CNTK):

first minibatch:
```
Epoch[ 4 of 5]-Minibatch[   1-  10, 0.0000000000003%]
```
...
last minibatch before Epoch completes:
```
Epoch[ 4 of 5]-Minibatch[51471-51480, 0.0000000015628%] 
```
when training is started with makeMode = false, percentages are displayed correctly (0%-100%).

-----------------------------

**2) Display issue - learning rate 0**
```
Starting Epoch 4: learning rate per sample = 0.000000  effective momentum = 0.899967  momentum as time constant = 13283.1 samples
```
Please modify to use the floating point - otherwise in large minibatches 0 is displayed (and logged)

-----------------------------

**3) Regression issue?**

Seems that autoAdjust = [] section is no longer executed when makeMode = true, Is that expected behavior?

With re-training the model from scratch (makeMode = false), the LR is adjusted into Epochs 4 and 5, unlike what can be seen with makeMode = true continuing from Epoch 4. Is the model/checkpoint properly loaded at restart?

**Comment by ivrodr-msft** [comment]
The first issue was already addressed in this commit: [https://github.com/Microsoft/CNTK/commit/8cb17fa526844bb0c2b8ea4c4a147f809d4be621](commit8cb17fa).
